onnx/backend/test/data/model/test_GLU/model.pb

```diff
@@ -1,4 +1,4 @@
-pytorch0.2"com.facebook:�
+pytorch0.3:�
```

Comment:
@ezyang @bddppq now, it's 0.3.

Reply:
Yeah, we need a better story for getting PyTorch's version number to the ONNX exporter. Probably best to make it another arg so you don't have a compile time constant.